src/lib/containers/table/SynapseTableUtils.ts

```diff
@@ -0,0 +1,166 @@
+import { lexer } from 'sql-parser'
+import {
+  formatSQLFromParser,
+  isGroupByInSql,
+} from '../../utils/functions/sqlFunctions'
+import { ColumnType, QueryResultBundle, Row } from '../../utils/synapseTypes'
+
+type ColumnReference = {
+  index: number
+  name: string
+}
+
+export const getColumnIndiciesWithType = (
```

Comment:
Everything in this file was moved here. No new code other than some light refactoring just to convert component instance methods to be standalone functions